cms/djangoapps/cms_user_tasks/tests.py

```diff
@@ -79,6 +78,17 @@ def setUpTestData(cls):
             total_steps=5)
         cls.artifact = UserTaskArtifact.objects.create(status=cls.status, text='Lorem ipsum')
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.app = Celery('user_tasks', broker='memory://localhost/')
```

Comment:
@jmbowman as `@override_settings` not working, should we change broker of entire cms test suite to `memory` or should we do something like this only for this tests

Reply:
I'm fine with changing it for the whole test suite; the settings default to using RabbitMQ which we actually don't use anywhere now (it's for backwards compatibility in Open edX installations), so the memory broker is a more useful default for the unit tests anyway.

Reply:
Thanks. I will update it accordingly